### PR TITLE
Migrate PowerUtils calls to PowerManagementService

### PR DESCRIPTION
**Pull request by ezaquarii**
Signed-off-by: Chris Narkiewicz <hello@ezaquarii.com>

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -284,8 +285,8 @@ dependencies {
     implementation 'com.google.dagger:dagger:2.22.1'
     implementation 'com.google.dagger:dagger-android:2.22.1'
     implementation 'com.google.dagger:dagger-android-support:2.22.1'
-    annotationProcessor 'com.google.dagger:dagger-compiler:2.22.1'
-    annotationProcessor 'com.google.dagger:dagger-android-processor:2.22.1'
+    kapt 'com.google.dagger:dagger-compiler:2.22.1'
```
Comment (ezaquarii) [inline build.gradle]: `kapt` does the same as `annotationProcessor` except it supports Kotlin as well. This particular change enabled Kotlin Dagger modules.

**Comment by ezaquarii** [comment]
@tobiasKaminsky
```
INSTRUMENTATION STATUS: stack=java.lang.NullPointerException: Attempt to invoke virtual method 'void android.widget.TextView.setText(java.lang.CharSequence)' on a null object reference
```

Looks odd. Fails despite restarts. Any similar issues in the past?

File: build.gradle (hunk 1 above)
Comment (AndyScherzinger) [reply build.gradle]: will need a rebase soon. ATM I am working on getitng all the dependabot changes merged to master (in this case 2.23.1)

Comment (ezaquarii) [reply build.gradle]: ack

Comment (AndyScherzinger) [reply build.gradle]: DONE ✔️

Comment (ezaquarii) [reply build.gradle]: Conflict resolved.

**Comment by ezaquarii** [comment]
@tobiasKaminsky Conflict resolved. Ready to merge.

**Pull request merged**